src/ruler.js

```diff
@@ -91,12 +109,19 @@ export class DragRulerRuler extends Ruler {
 		this.labels.removeChildren().forEach(c => c.destroy());
 	}
 
-	dragRulerDeleteWaypoint(event={preventDefault: () => {return}}) {
+	dragRulerDeleteWaypoint(event={preventDefault: () => {return}}, options={snap: true}) {
```

Comment:
This will fail to set `snap: true` as default if options without `snap` in them are passed. Better use an empty object as default and then add `options.snap = options.snap ?? true` to the code below to set that value if it's unset